tools/scripts/notes.md

```diff
@@ -11,14 +11,28 @@ What I imagine for this script in the future:
 
 ## Missing checks?
 
-### Enable remote file shares (client and server)
+### Enable admin remote file shares (client and server)
+
+*Notice: Restart required for changes to take effect.*
 
 Nessus needs access to remote file shares like C$ and ADMIN$.
 
 Set AutoShareServer and AutoShareWks in `HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\LanmanServer\Parameters` to 1 or remove them.
 
+```ps
+Get-SmbServerConfiguration | Select AutoShareServer,AutoShareWorkstation
+
+# Enable
+Set-SmbServerConfiguration -AutoShareServer $True -AutoShareWorkstation $True -Confirm:$false
+
+# Disable
+Set-SmbServerConfiguration -AutoShareServer  $False -AutoShareWorkstation $False -Confirm:$false
+```
+
 ### Network logons
 
+<!--Only for Windows XP?-->
```

Comment:
```suggestion
*Notice: This setting is only required when scanning Windows XP*
```
Is there an explanation on the why available somewhere?

Reply:
[Here](https://github.com/kAh00t/Nessus-Powershell-Oneliners/blob/main/NessusOneLiners.md#confirm-forceguest-is-set-to-0-classic-is-required-for-nessus-seemingly---windows-xp-only-see-localaccounttokenfilterpolicy-for-win-7-and-above)

Windows 7 and above use LocalAccountTokenFilterPolicy.

Reply:
```suggestion
*Notice: This setting is only required when scanning Windows XP. Scans for Windows 7 and above use `LocalAccountTokenFilterPolicy`.[^force-guest]*
```